packages/module/src/QuickStartPanelContent.tsx

```diff
@@ -98,10 +101,12 @@ const QuickStartPanelContent: React.FC<QuickStartPanelContentProps> = ({
             <Title
               headingLevel="h1"
               size="xl"
+              className="pfext-quick-start-panel-content__name"
               style={{ marginRight: 'var(--pf-global--spacer--md)' }}
             >
               {quickStart?.spec.displayName}{' '}
-              <small className="pfext-quick-start-panel-content__duration text-secondary">
+              <small className="pfext-quick-start-panel-content__duration">
+                Quick start •{' '}
```

Comment:
1. Should this text always say `Quick start` ?
2. There should be a new resource defined that includes this text